From 1522da763a3dde5cbf6657e873fee2d71b6abf15 Mon Sep 17 00:00:00 2001
Message-ID: <1522da763a3dde5cbf6657e873fee2d71b6abf15.1739309136.git.roman@burningswell.com>
From: Roman Scherer <roman@burningswell.com>
Date: Tue, 11 Feb 2025 22:23:37 +0100
Subject: [PATCH] gnu: forges: Use %current-system instead of "x86_64-linux".

* src/cuirass/forges.scm (%default-jobset-options-systems): Use
%current-system instead of "x86_64-linux".
---
 src/cuirass/forges.scm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/cuirass/forges.scm b/src/cuirass/forges.scm
index 56f4876..540315b 100644
--- a/src/cuirass/forges.scm
+++ b/src/cuirass/forges.scm
@@ -17,6 +17,7 @@
 ;;; along with Cuirass.  If not, see <http://www.gnu.org/licenses/>.
 
 (define-module (cuirass forges)
+  #:use-module ((guix utils) #:select (%current-system))
   #:use-module (cuirass specification)
   #:use-module (json)
   #:use-module (ice-9 match)
@@ -48,7 +49,7 @@
 
 ;; Default target systems for jobsets created using a forge module.
 (define %default-jobset-options-systems
-  (list "x86_64-linux"))
+  (list (%current-system)))
 
 ;; This mapping defines a specific JSON dictionary used for tweaking Cuirass
 ;; options. It is not included in the JSON data sent by default by Gitlab and
-- 
2.48.1

